cmd/jimmctl/cmd/relation.go

```diff
@@ -84,85 +84,69 @@ If target_object is an application offer, the relation can be one of:
 
 
 Additionally, if the object is a group, a userset can be applied by adding #member as follows.
-This will grant/revoke the relation to all users within TeamA:
+This will grant/revoke access to all users within TeamA:
 
 	group-TeamA#member administrator controller-MyController
 `
 
-	addRelationDoc = `
-The add command adds relation to jimm.
+	addPermissionDoc = `
+Adds access to a resource.
```

Comment:
Again we say addPermissionDoc, but then it says Adds access to a resource. It's also more "granting" rather than adding right? I'd go with sometihng more like Juju, they explain what you can grant to in their grant doc:
```
Grants access level to a Juju user for a model, controller, or application offer.
```

So when we settle on permission or access, let's say "<Grants/Adds> <permission/access> level to a JIMM user for a model, etc, etc,

Reply:
I also don't really mind which, but just some consistency and caution as I think this will end up in stone honestly.